webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -746,6 +746,7 @@ class TechniqueArchiverImpl (
     val filesToAdd = (
       "metadata.xml" +:
       "technique.cf" +:
+      "rudder_reporting.cf" +:
```

Comment:
can you sort them alpha-num? (or at least let the technique* together at the end?)

Reply:
It can be self merged afterward